tests/data/search_result_20200130_093253_ssc6_u0001.json

```diff
@@ -0,0 +1,497 @@
+                    26.702710686472205
+                ],
+                [
+                    57.100995519928176,
+                    26.702710875510398
+                ],
+                [
+                    57.10109746070303,
+                    26.695602603249792
+                ],
+                [
+                    57.1010171847999,
+                    26.695603543459217
+                ],
+                [
+                    57.10110269632445,
+                    26.688516865680906
+                ],
+                [
+                    57.101031973972475,
+                    26.688517741834684
+                ],
+                [
+                    57.101138719473916,
+                    26.68139672247875
+                ],
+                [
+                    57.101036413155434,
+                    26.68139811849534
+                ],
+                [
+                    57.101146337488906,
+                    26.674279062623775
+                ],
+                [
+                    57.08012191931233,
+                    26.674581777360412
+                ],
+                [
+                    57.08018320144918,
+                    26.669329526837064
+                ],
+                [
+                    57.08010245611476,
+                    26.669330333596545
+                ],
+                [
+                    57.080068248959506,
+                    26.662270637540992
+                ],
+                [
+                    57.08001787883328,
+                    26.662271158826123
+                ],
+                [
+                    57.080048784884546,
+                    26.655158426164014
+                ],
+                [
+                    57.08000650261795,
+                    26.655158892026947
+                ]
+            ]
+        ],
+        "type": "Polygon"
+    },
+    "id": "20200130_093253_ssc6_u0001",
+    "properties": {
+        "acquired": "2020-01-30T09:32:53.187Z",
+        "availability": "2020-01-30T09:32:53.187Z",
+        "clear_confidence_percent": 96,
+        "clear_percent": 99,
+        "cloud_cover": 0,
+        "cloud_percent": 0,
+        "ground_control_ratio": 0.87,
+        "gsd": 0.72,
+        "heavy_haze_percent": 0,
+        "item_type": "SkySatCollect",
+        "light_haze_percent": 0,
+        "pixel_resolution": 0.5,
+        "provider": "skysat",
+        "published": "2020-01-31T13:40:08Z",
+        "publishing_stage": "finalized",
+        "quality_category": "standard",
+        "satellite_azimuth": 95.9,
+        "satellite_id": "SSC6",
+        "shadow_percent": 0,
+        "snow_ice_percent": 0,
+        "strip_id": "s106_20200130T093253Z",
+        "sun_azimuth": 202.2,
+        "sun_elevation": 42.6,
+        "updated": "2020-01-31T13:40:08Z",
+        "view_angle": 15.8,
+        "visible_confidence_percent": 89,
+        "visible_percent": 100
+    },
+    "type": "Feature"
+}
```

Comment:
May want to make the editors happy by adding the newline here

Reply:
where should the newline go?

Reply:
At the end of the doc. It doesn't show it in the commends, but if you look at the diff you will see the red circle and the hover message indicates the issue is no newline.

![image](https://user-images.githubusercontent.com/2984664/170563585-b8de87b0-42b7-477e-abfd-38ca563d65b4.png)

Reply:
code updated!